vulcanai2/tests/models/test_utils.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import torch
+from vulcanai2.models.utils import (round_list,
+                                   get_one_hot,
+                                   pad,
+                                   set_tensor_device,
+                                   master_device_setter)
+
+def test_round_list():
+    """Test if the list is rounded to the desired
+    decimals"""
+    test_list = [0.83754245, 0.13249807]
+    out = round_list(test_list, decimals=3)
+    assert len(str(out[0]).split(".")[1]) == 3
+
+def test_get_one_hot():
+    """Test for get_one_hot"""
+    test_inp = np.array([0,1,2,3,4,3,2,1,0])
+    n_classes = 5
+    assert get_one_hot(test_inp).shape == (len(test_inp),
```

Comment:
would you be able to actually handcode the one-hot matrix and see if it returns the same thing? might be the simplest and most robust test for this.